pyproject.toml

```diff
@@ -0,0 +1,113 @@
+[build-system]
+build-backend = "setuptools.build_meta:__legacy__"
```

Comment:
Currently need to use legacy since versioneer is vendored ( https://github.com/python-versioneer/python-versioneer/issues/249#issuecomment-1235845640 ). Though we could devendor versioneer